designer/client/hooks/list/useListItem/useListItem.tsx

```diff
@@ -13,6 +13,7 @@ export function useListItem(state, dispatch): ListItemHook {
   const { value = "", condition } = selectedItem;
 
   function handleTitleChange(e) {
+    selectedItem.value = e.target.value;
     dispatch({
       type: ListActions.EDIT_LIST_ITEM_TEXT,
       payload: e.target.value,
```

Comment:
This should behave like the page edit page (when you change the title, the path changes too).

So the best way to achieve this is to add this to the EDIT_LIST_ITEM_TEXT action. You also have to be careful of overriding existing values (because someone may have set it purposefully previously). Perhaps it should only autofill when it is a new list item?

Reply:
@jenbutongit Bit confused now. As far as i can see pageEdit does this in a similar way i'm doing it (A little bit more to it)? I do agree that It probably should autofill when it is a new list.